src/main/java/com/jezhumble/javasysmon/LinuxMonitor.java

```diff
@@ -73,6 +77,16 @@ public MemoryStats physical() {
         return new MemoryStats(free, total);
     }
 
+    public MemoryStats physicalWithBuffersAndCached(){
+        String totalMemory = fileUtils.runRegexOnFile(TOTAL_MEMORY_PATTERN, "/proc/meminfo");
```

Comment:
I am thinking that maybe we can also extract the hard-coded literal `"/proc/meminfo"` to a constant, maybe `MEMINFO_FILE `or something similar.